srcs/utils_ray.c

```diff
@@ -0,0 +1,106 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   utils_ray.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: tgrivel <marvin@42lausanne.ch>             +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/09/20 05:43:03 by tgrivel           #+#    #+#             */
+/*   Updated: 2022/09/20 15:16:00 by melogr@phy       ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include	"cub3d.h"
+
+// For tile top left, we need to reduce of a bit
+#define CASE_NEG	0.00001f
+
+// Collision with rising horizontal line
+float	horizontal_up(t_info *info, float hit[2], float dx, float dy)
```

Comment:
See prototype change suggestion
```suggestion
float	horizontal_up(t_info *info, float hit[2], float delta[2])
```

Reply:
Done: eee14faa5865180a8d97b2d8fce6f10eaf627448